src/main/java/chronologer/command/ExitCommand.java

```diff
@@ -1,30 +1,24 @@
 package chronologer.command;
 
-import chronologer.exception.ChronologerException;
 import chronologer.storage.Storage;
 import chronologer.task.TaskList;
-import chronologer.ui.UiTemporary;
 
 /**
  * Terminates the program.
  *
  * @author Sai Ganesh Suresh
- * @version v1.3
+ * @version v1.4
  */
 public class ExitCommand extends Command {
 
-    private static final String GOODBYE = "Goodbye";
-
     public ExitCommand() {
     }
 
     /**
      * Terminates the program by updating isExit to true.
      */
-    public void execute(TaskList tasks, Storage storage) throws ChronologerException {
-        super.commandOut();
-        UiTemporary.printGoodbye();
-        UiTemporary.printOutput(GOODBYE);
+    public void execute(TaskList tasks, Storage storage)  {
+        System.exit(0);
```

Comment:
maybe reduntant, but should we retain the UiTemporary printout and have the System.exit(0) line appended after it?

Reply:
It won't show up, I tried. The system exits faster than it prints. You can pause using a thread function but then it throws a different exception which will affect the parent command class so I decided to left it out for now.

Reply:
ok we'll try figure something out if we need to retain the exit message later on

Reply:
Ah dont worry i have a quick save feature done up! should be fine!